src/js/core.js

```diff
@@ -873,6 +873,16 @@
             }
         },
 
+        clearSelection: function (toEnd) {
```

Comment:
Because its implementation is different? `Selection.collapse` expects `parentNode` and `offset` parameters, whilst this method is a shortcut / switch between `collapseToStart` and `collapseToEnd`